src/services/test-helpers/mock/data/foreignAssets.ts

```diff
@@ -0,0 +1,12 @@
+// '{"parents":"1","interior":{"X2":[{"Parachain":"2125"},{"GeneralIndex":"0"}]}}';
```

Comment:
Do we still need this comment? Also, Im thinking we should add the copyright to this file. I see a few other mock data files that we missed the copyright for as well that we can update at some point.

Reply:
> in some other files that the copyright was with hear 2017-2022 I updated it to 2017-2023 but not all.

Yea lets try to save the rest for a seperate PR, it just adds a lot of bloat to the review.